common/src/main/webapp/sass/datepicker/datepicker.scss

```diff
@@ -5,25 +5,23 @@
   display: inline-block;
   padding: 0;
   border: 0;
-  width: 100%;
 }
 
 .react-datepicker {
   font-family: $datepicker__font-family;
   font-size: $datepicker__font-size;
-  background-color: var(--date-picker-background-color);
```

Comment:
Is `--date-picker-background-color` not used anymore? Should we remove it from wherever it's defined?